dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/BaseDimensionalItemObject.java

```diff
@@ -59,6 +59,12 @@
      */
     protected AggregationType aggregationType;
 
+    /**
+     * A value representing a period offset that can be applied to Dimensional Item
+     * Object within a Indicator formula
+     */
+    protected int periodOffset = 0;
```

Comment:
Is this a "transient" property and could be made transient just to indicate that?

Reply:
Fixed (add `transient`0